src/components/common/Inputs/input.js

```diff
@@ -1,9 +0,0 @@
-import React from "react";
-
-const Input = props => (
```

Comment:
@abayo-luc, Why you did you delete the shared components and created your own? What do you think if you put it back and add what we have missed while building it?

Reply:
+1 to that.